### PR TITLE
feat: npm package root path

### PR DESCRIPTION
**Pull request by Anush008**
## Description
This PR intends to make the `pkgRoot` value of the `@semantic-release/npm` plugin configurable which previously had a default value of `"."`.

This enables  us to publish `npm` packages from sub-directories of a repo. For use in resolving https://github.com/open-sauced/pizza-cli/issues/27.

A demo release using this configuration can be found here.
https://github.com/Anush008/xdg/releases/tag/v1.10.0

The config.
https://github.com/Anush008/xdg/blob/2bcf6c2d7121ed8f77a0c31679458052fbc8b74b/.github/workflows/release.yaml#L29-L35

The published sub-directory: https://www.npmjs.com/package/xdg-anush

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Related to https://github.com/open-sauced/pizza-cli/issues/27.

## Mobile & Desktop Screenshots/Recordings
N/A

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [X] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Anush008** [comment]
cc @0-vortex.

**Review by 0-vortex — APPROVED** [review]
LGTM! 👍

**Review by bdougie — APPROVED** [review]

File: release.config.js [hunk 1]
```diff
@@ -170,6 +170,8 @@ addPlugin("@semantic-release/git", {
     "CHANGELOG.md",
     "package.json",
     "package-lock.json",
+    "**/package.json",
```
Comment (0-vortex) [inline release.config.js]: To include all package managers this should work:

```
"package.json",
"package-lock.json",
"npm-shrinkwrap.json",
"yarn.lock",
"pnpm-lock.yaml",
"**/package.json",
"**/package-lock.json",
"**/npm-shrinkwrap.json",
"**/yarn.lock",
"**/pnpm-lock.yaml",
```

**Review by 0-vortex — CHANGES_REQUESTED** [review]
Couple of minor changes you caught in git! 🚀 🍕

File: release.config.js (hunk 1 above)
Comment (Anush008) [reply release.config.js]: https://github.com/open-sauced/release/commit/73b8c0d70db730b91a4b7c99ab0ad88ab2bdc9c2

**Review by jpmcb — APPROVED** [review]

**Pull request merged**